portal-web/test/functional/com/liferay/portalweb/macros/analyticscloud/ACApi.macro

```diff
@@ -126,6 +169,17 @@ definition {
 		return ${token};
 	}
 
+	macro searchApiResponse {
+		if (!(isSet(resultPosition))) {
+			var searchedValue = JSONUtil.getWithJSONPath(${apiDataResponse}, "$.${jsonQuery}");
+		}
+		else {
+			var searchedValue = JSONUtil.getWithJSONPath(${apiDataResponse}, "$.results[${resultPosition}]${jsonQuery}");
```

Comment:
Hey @manuferreira, I think you could use `assertGenericJSONPathResponse` macro that search in the result of the API response and then checks the result you expect

I don't think you need to use `$.results` to navigate to what's inside results, if you use `$..` it will work too which is what the macro uses. What do you think?

Reply:
I'm seeing that I created this macro assertGenericJSONPathResponse lol :rofl: